tests/Microsoft.Identity.Test.Unit.net45/CoreTests/CacheTests/MsalTokenCacheKeysTests.cs

```diff
@@ -26,12 +26,10 @@
 //------------------------------------------------------------------------------
 
 using System;
-using System.Linq;
-using Microsoft.Identity.Client.Cache;
 using Microsoft.Identity.Client.Cache.Keys;
-using Microsoft.Identity.Test.Common;
 using Microsoft.Identity.Test.Common.Core.Helpers;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Microsoft.Identity.Client.Cache.Keys.MsalCacheKeys;
```

Comment:
what does "using static" do?

Reply:
Sorry I was a bit lazy and took VS's first suggestion. It allows you to write "X" instead of "MsalCacheKeys.X".